plugins/kvscheduler/internal/utils/record.go

```diff
@@ -72,21 +74,21 @@ func (p *RecordedProtoMessage) UnmarshalJSON(data []byte) error {
 	if p.ProtoMsgName == "" {
 		return nil
 	}
-	msgType := proto.MessageType(pwn.ProtoMsgName)
-	if msgType == nil {
-		return fmt.Errorf("unknown proto message: %s", p.ProtoMsgName)
-	}
-	msg := reflect.New(msgType.Elem()).Interface().(proto.Message)
-	var err error
-	if len(pwn.ProtoMsgData) > 0 && pwn.ProtoMsgData[0] == '{' {
-		err = jsonpb.UnmarshalString(pwn.ProtoMsgData, msg)
-	} else {
-		err = proto.UnmarshalText(pwn.ProtoMsgData, msg)
-	}
-	if err != nil {
-		return err
-	}
-	p.Message = msg
+	/*msgType := proto.MessageType(pwn.ProtoMsgName)
+	  if msgType == nil {
+	  	return fmt.Errorf("unknown proto message: %s", p.ProtoMsgName)
+	  }
+	  msg := reflect.New(msgType.Elem()).Interface().(proto.Message)
+	  var err error
+	  if len(pwn.ProtoMsgData) > 0 && pwn.ProtoMsgData[0] == '{' {
+	  	err = jsonpb.UnmarshalString(pwn.ProtoMsgData, msg)
+	  } else {
+	  	err = proto.UnmarshalText(pwn.ProtoMsgData, msg)
+	  }
+	  if err != nil {
+	  	return err
+	  }
+	  p.Message = msg*/
```

Comment:
Same here

Reply:
Also godoc for this function need to be updated as no jsonpb is used.